src/App.jsx

```diff
@@ -23,10 +23,15 @@ const App = () => {
       limitCount: "",
       expiration: "",
     },
+    optional: {
```

Comment:
Optional comment and amount of keys are two different things. Take properties outside of optional object.

Reply:
done